manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml

```
@@ -215,6 +231,10 @@ spec:
             postgresqlCpuRequest:
               description: 'PostgreSQL deployment CPU request (default: no request)'
               type: string
+            postgresqlImage:
+              description: Image string used for the postgresql deployment By default
+                PostgresqlImageName:PostGresqlImageTag takes effect
```

Typo
```suggestion
                PostgresqlImageName:PostgresqlImageTag takes effect
```

(There are other locations below with a similar typo)